dvc/command/experiments.py

```diff
@@ -474,6 +463,17 @@ def _format_json(item):
 
 
 class CmdExperimentsShow(CmdBase):
+    @staticmethod
+    def _show_csv(td):
+        _post_process_td(td)
+        ui.write(td.to_csv())
```

Comment:
Also, `to_csv()` already adds a `\r\n`, so, there'll be two empty lines at the end, one from csv, and other from `ui.write`.
You may want to use `ui.write(..., end="")`.